src/lib/components/Button.react.js

```diff
@@ -0,0 +1,153 @@
+// @flow
+
+import React, { Component } from 'react';
+import MUIButton from '@material-ui/core/Button';
+import MUIIconButton from '@material-ui/core/IconButton';
+
+type Props = {
+  /**
+   * This is what will be displayed inside the button.
+   * If a label is specified, the text within the label prop will
+   * be displayed. Otherwise, the component will expect children
+   * which will then be displayed. (In our example,
+   * we are nesting an `<input type="file" />` and a `span`
+   * that acts as our label to be displayed.) This only
+   * applies to flat and disableShadow buttons.
+   */
+  children?: Node,
+  /** Mapping from MUI Element name to the intended classname for that Element
+   * See https://material-ui.com/api/button/#css
+   * and https://material-ui.com/api/icon-button/#css*/
+  classes?: Object,
```

Comment:
Can replace `Object` with
```js
{
  root?: string,
  ...
}
```

See https://github.com/StratoDem/sd-material-ui/pull/427/files#r471462161

Reply:
Got it, I'll change this for all of the PR's. How does the style look wrong?

Reply:
![image](https://user-images.githubusercontent.com/16123745/90399606-38f52800-e069-11ea-9977-f3d604359442.png)

The raised/flat button labels are not centered vertically

Reply:
I think it's just cut off, here's a better view
![image](https://user-images.githubusercontent.com/47650493/90401362-d94c4c00-e06b-11ea-83fa-b04660c51855.png)

Reply:
That's a better view of the problem. The button label is not vertically centered within the button, even though it should be.

Reply:
![image](https://user-images.githubusercontent.com/47650493/90411058-34387000-e079-11ea-96b0-c98b37edfba0.png)
Turns out the problem was just the margin of the children...